lib/encoders/date_time.ex

```diff
@@ -0,0 +1,5 @@
+defimpl Poison.Encoder, for: Ecto.DateTime do
+  def encode(date_time, options) do
+    date_time |> Ecto.DateTime.to_string |> Poison.encode!(options)
```

Comment:
Ah! I didn't realize there was a new method. I'll add that. Finishing up the tests right now actually :)